### DISCUSSION TITLE
How to use Each command `direction` attribute in Excel markup

### DISCUSSION THREAD

**Discussion by allardq** [jxlsteam/jxls#201] [Q&A] [answered]
Hi! I understand from the documentation (http://jxls.sourceforge.net/reference/each_command.html) that we can change the direction in which the values repeat (either `DOWN` or `RIGHT`) and there is a Java example which shows how this is done.

I would like to do the same in Excel markup, and have tried the following:
1. `direction=EachCommand.Direction.RIGHT`
2. `direction="EachCommand.Direction.RIGHT"`
3. `direction="RIGHT"`
4. `jx:each(...).setDirection(EachCommand.Direction.RIGHT)`

but none of these options has worked so far. What should the syntax be?

**Comment by SoltauFintel** [comment, marked as answer]
Here's an example and I have updated the documation: `jx:each(items="salaries" var="s" direction="RIGHT" lastCell="F2")`

**Reply by allardq** [reply]
Got it, thank you! I believe my issue was that I forgot to change the `lastCell` value of the `jx:area` which caused the change in direction to not be observed.
